integration_tests/src/test/scala/com/nvidia/spark/rapids/tests/tpch/TpchLikeSparkSuite.scala

```diff
@@ -17,19 +17,26 @@
 package com.nvidia.spark.rapids.tests.tpch
 
 import com.nvidia.spark.RapidsShuffleManager
-import com.nvidia.spark.rapids.ColumnarRdd
+import com.nvidia.spark.rapids.{ColumnarRdd, ExecutionPlanCaptureCallback}
 import org.scalatest.{BeforeAndAfterAll, FunSuite}
 
-import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.sql.execution.adaptive.AdaptiveSparkPlanExec
 
 class TpchLikeSparkSuite extends FunSuite with BeforeAndAfterAll {
 
-  lazy val  session: SparkSession = {
+  val adaptiveQueryEnabled = Option(System.getProperty("spark.sql.adaptive.enabled"))
```

Comment:
Rather than a system property, shouldn't this be a parameterized test to run it both with and without AQE?

Reply:
The tests are now parameterized.